staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/micro_time.go

```diff
@@ -175,10 +175,10 @@ func (t *MicroTime) Fuzz(c fuzz.Continue) {
 	if t == nil {
 		return
 	}
-	// Allow for about 1000 years of randomness.  Leave off nanoseconds
-	// because JSON doesn't represent them so they can't round-trip
-	// properly.
-	t.Time = time.Unix(c.Rand.Int63n(1000*365*24*60*60*1000*1000), 0)
```

Comment:
so we had 1000*1000*1000 years randomness before, right?

Reply:
Yes.
https://golang.org/pkg/time/#Unix

Reply:
It said:
> Not all sec values have a corresponding time value. One such value is 1<<63-1 (the largest int64 value).

Reply:
The change looks right